guides/source/configuring.md

```diff
@@ -969,6 +969,10 @@ Define an `Array` specifying the key/value tags to be inserted in an SQL
 comment. Defaults to `[ :application ]`, a predefined tag returning the
 application name.
 
+#### `config.active_record.query_log_tags_format`
+
+A `Symbol` specifying the formatter to use for tags. Valid values are `[:sqlcommenter, :default]`.  Defaults to `:default`.
```

Comment:
Nit:
```suggestion
A `Symbol` specifying the formatter to use for tags. Valid values are `:sqlcommenter` and `:default`.  Defaults to `:default`.
```

I think the array syntax makes it less clear that the individual symbols are the valid values